log/log.go

```diff
@@ -55,18 +53,21 @@ func init() {
 	encoderConf.EncodeTime = func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
 		enc.AppendString(t.Format(time.RFC3339))
 	}
+	encoderConf.CallerKey = "source"
 
 	l := zap.New(
 		zapcore.NewCore(
 			zapcore.NewJSONEncoder(encoderConf),
 			zapcore.Lock(os.Stdout),
 			zap.NewAtomicLevel(),
-		))
+		),
+		zap.AddCaller(),
+		zap.AddCallerSkip(1),
+		zap.AddStacktrace(zapcore.ErrorLevel),
```

Comment:
Vad innebär detta? Skulle det inte vara bra se annars också (iaf vid warning)?

Reply:
Det är standard från `zap` med stacktrace vid error, dock så försvann den när vi lade in RFC3339 timestamp.
Ändrar till info